src/GovUk.Education.ExploreEducationStatistics.Admin/Controllers/Api/Security/PermissionsController.cs

```diff
@@ -116,18 +122,32 @@ public Task<ActionResult<bool>> CanUpdateMethodology(Guid methodologyId)
                 _userService.CheckCanUpdateMethodologyVersion);
         }
 
-        [HttpGet("permissions/methodology/{methodologyId:guid}/status/draft")]
-        public Task<ActionResult<bool>> CanMarkMethodologyAsDraft(Guid methodologyId)
-        {
-            return CheckPolicyAgainstEntity<MethodologyVersion>(methodologyId,
-                _userService.CheckCanMarkMethodologyVersionAsDraft);
-        }
+        public record MethodologyApprovalStatusPermissions(
+            bool CanMarkDraft, bool CanMarkHigherLevelReview, bool CanMarkApproved);
 
-        [HttpGet("permissions/methodology/{methodologyId:guid}/status/approve")]
-        public Task<ActionResult<bool>> CanApproveMethodology(Guid methodologyId)
+        [HttpGet("permissions/methodology/{methodologyVersionId:guid}/status")]
+        public async Task<ActionResult<MethodologyApprovalStatusPermissions>> CanUpdateMethodologyApprovalStatus(
```

Comment:
I'm guessing this is named this way due to other conventions, but I would've maybe expected the name to be more like "GetMethodologyApprovalPermissions"

Reply:
I've renamed it.